lib/runner/request-helpers-presend.js

```diff
@@ -189,7 +189,7 @@ module.exports = [
             return done();
         }
 
-        authInterface = createAuthInterface(auth, context.protocolProfileBehavior);
+        authInterface = createAuthInterface(auth, context.protocolProfileBehavior, run.options.requester);
```

Comment:
Maybe we can just pass the specific things from `options.requester` to keep the interface restrictive?